.eslintrc.js

```diff
@@ -6,33 +6,30 @@ module.exports = {
 		ecmaVersion: 2017,
 		sourceType: 'module'
 	},
-	plugins: [
-		'ember'
-	],
 	extends: [
 		'eslint-config-bbva'
 	],
 	env: {
-		browser: true
+		browser: false,
+		node: true
 	},
+	plugins: ['node'],
+	rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
```

Comment:
You can extend from node instead of override this: `"extends": ["plugin:node/recommended", "eslint-config-bbva"], `

Reply:
Done